app/models/bulkrax/csv_entry.rb

```diff
@@ -70,9 +70,10 @@ def self.data_for_entry(data, _source_id, parser)
 
     def build_metadata
       raise StandardError, 'Record not found' if record.nil?
```

Comment:
As I'm thinking on this more, I think it would also make sense to extract this guard condition to it's own method.  The rest of this method is calls to other methods; which makes the raise errors section a bit of an outlier.  It also means that in https://github.com/scientist-softserv/palni-palci/pull/428 you needed to override the entire method.

Reply:
totally agree! I was actually thinking along the same lines yesterday when i was looking at this. I have refactored everything now and I think it's a lot better, but I'm open to any other changes you might suggest.